ci-operator/step-registry/compliance/cve-agent/report/compliance-cve-agent-report-commands.sh

```
@@ -0,0 +1,234 @@
+ls -la "${SHARED_DIR}/" 2>/dev/null || echo "(empty)"
+echo ""
+
+# Collect analysis artifacts (stored as flat files with cve-artifact-<CVE_ID>- prefix)
+REPORT_ARTIFACT=""
+ARTIFACT_PREFIX="cve-artifact-${CVE_ID}-"
+ARTIFACT_COUNT=0
+echo "Looking for artifacts with prefix: ${ARTIFACT_PREFIX}"
+for f in "${SHARED_DIR}/${ARTIFACT_PREFIX}"*; do
+  [ -f "$f" ] || continue
+  BASENAME=$(basename "$f")
+  ORIGINAL_NAME="${BASENAME#${ARTIFACT_PREFIX}}"
```

Shellcheck (0.11.0)

[info] Expansions inside ${..} need to be quoted separately, otherwise they match as patterns.

(SC2295)